rulestest.sh

```diff
@@ -58,6 +63,12 @@ echo "Restarting $TARGET_CONTAINER"
 podman run \
     --name $TARGET_CONTAINER \
     --pod cryostat \
-    --env JMX_PORT=9093 \
-    --env USE_AUTH=true \
```

Comment:
Sorry, just noticed this now, but is getting rid of `USE_AUTH` intentional?

Reply:
See other comment